spec/features/organization/subjects/create_spec.rb

```diff
@@ -0,0 +1,49 @@
+require "rails_helper"
+
+feature "Create subject" do
+  before do
+    create(:user, :director, email: "example@gmail.com", organization: organization)
```

Comment:
NBD: Here we can use Faker - `Faker::Internet.email`

Reply:
I think i can just remove it)